docs/README.building

```diff
@@ -0,0 +1,119 @@
+FreeRDP has a few dependencies that are required for proper operation:
+
+1. SSL
+
+RDP requires a secure tunnel and utilizes TLS for this. We do not implement this
+ourselves but reuse existing libraries:
+
+We support
+
+* OpenSSL our main development SSL library
+* LibreSSL (supported by community)
+* MBedTLS (supported by community)
+
+2. Kerberos
+
+Authentication to many services requires kerbers (especially if smartcards are in use)
+
+We support:
+
+* MIT
+* Heimdal
+
+3. JSON
+
+Azure logon requires HTTP/JSON messages to be parsed.
+
+We support:
+
+* cJSON
+
+4. H264
+
+RDP GFX modes (anything newer Windows 8.1 / Server 2012) supports a graphics mode based
+on the H264 codec
+
+We support
+
+* OpenH264
+* FFMPEG (x264 or OpenH264)
+
+There are some platform specific implementations too (e.g. mediacodec on android) but these
+two are the options that are always required.
+
+5. Graphics scaling support
+
+High DPI support and smart-sizing option require bitmaps to be scaled by the client. 
+
+We support
+
+* Swscale
+* Cairo
+
+6. Audio encoders/decoders
+
+Sound and Microphone options allow transmission encoded in compressed formats. While also
```

Comment:
"allow transmission encoded in compressed formats" - looks like there is something missing here i.e. "allow transmission of audio samples encoded in compressed formats".

I'd also remove the `While also` from the next sentence.

Reply:
rephrased